docs/source/tutorial/tutorial-paginate-results.mdx

```diff
@@ -94,6 +94,14 @@ func loadMoreLaunchesIfTheyExist() {
 }
 ```
 
+Lastly, update the `init()` method to pass `cursor: nil` to `LaunchListQuery`:
+
+```swift title="LaunchListViewModel.swift"
+init() {
+    Network.shared.apollo.fetch(query: LaunchListQuery(cursor: nil)) { result in// highlight-line
```

Comment:
There is some guidance in the section for executing your first query after adding this code to the `init()` method that says to remove it since its only meant to be an initial illustration of running a query. If it's not clear I think we should update that section rather than continuing to reference this code.